plugins/scaffolder/src/plugin.tsx

```diff
@@ -134,6 +139,20 @@ export const EntityNamePickerFieldExtension = scaffolderPlugin.provide(
   }),
 );
 
+/**
+ * A field extension for selecting multiple entities that exists in the Catalog.
+ *
+ * @public
+ */
+export const MultiEntityPickerFieldExtension = scaffolderPlugin.provide(
```

Comment:
I think you'll also need to export this from the main `src/index.ts` for people to use. Looks like it might be missing.

You'll then also want to regenerate the api reports, which you can do by doing `yarn build:api-reports` and then commit the changes in markdown.